### DISCUSSION TITLE
Can I change YOURLS_URL_CONVERT value after install (w/Wo Links)?

### DISCUSSION THREAD

**Discussion by Akila3** [YOURLS/YOURLS#3273] [Q&A] [answered]
Can I change YOURLS_URL_CONVERT in the ..user/config.php value from 36 to 62 value after YOURLS was installed?
In the **FAQ** it mentioned not to change it if i have created links already.
But it doesn't mention if this could be changed if deleting all the links and recreating them again (using bulk import plugin) after the change is ok or not. or if no links are present, but YOURLS is fully installed.

Another reason why I am asking if it's ok to change after install is,  in the Bulk import plugin readme it states the following: 
"but remember that you shouldn't change this setting after installation." when it talks about the YOURLS_URL_CONVERT section in the readme.

So my question: is it safe to change this after install or not? even if i need to delete and re-create my links?

running on YOURLS v1.8.2

**Comment by ozh** [comment, marked as answer]
Going from 36 to 62 isn't problematic : you're enlarging the character set, you're allowing *more* chars to be used in short URLs

**Reply by Akila3** [reply]
thanks for your response.
what about in a case users wish to go from 62 to 36?
would deleting the links and recreating them again after the change is sufficient and safe to work?

**Reply by ozh** [reply]
I think old links would still function.

Of course deleting links and recreating would work. It's just the same as creating links. But then what's the point.
